ui/src/i18n/nl.js

```diff
@@ -0,0 +1,86 @@
+import deepmerge from 'deepmerge'
+import englishMessages from 'ra-language-dutch'
```

Comment:
You forgot to add this dependency to the `package.json`, and it is breaking the build.

You need to add it with: `npm install --save ra-language-dutch`